src/vs/workbench/contrib/chat/browser/chatContentParts/chatChangesSummaryPart.ts

```diff
@@ -130,7 +132,11 @@ export class ChatCheckpointFileChangesSummaryContentPart extends Disposable impl
 
 	private renderViewAllFileChangesButton(container: HTMLElement): IDisposable {
 		const button = container.appendChild($('.chat-view-changes-icon'));
+		this.hoverService.setupDelayedHover(button, () => ({
+			content: `View All File Changes`
+		}));
 		button.classList.add(...ThemeIcon.asClassNameArray(Codicon.diffMultiple));
+		button.tabIndex = 0;
```

Comment:
If this is actually a button, it should be `button` or use the aria role button. Otherwise it just looks like focusable text to screen readers.